DEVELOPMENT.md

```diff
@@ -126,3 +126,39 @@ On Fedora, this can be fixed by adding a `.conf` file to `/etc/sysctl.d/ increas
 fs.inotify.max_user_watches=2097152
 fs.inotify.max_user_instances=256
 ```
+
+### Error: can only create exec sessions on running containers: container state improper
+
+This may happen when the containers have stopped. This can be fixed by deleting the already created clusters and starting the script again.
+
+```bash
+sudo kind delete clusters us-east1 us-west1
+```
+
+### Failed to create kind cluster - "Reached target ._Multi-User System._|detected cgroup v1"
+
+This is happening probably due to [Docker Desktop 4.3.0+ using Cgroups v2](https://kind.sigs.k8s.io/docs/user/known-issues/#failure-to-create-cluster-with-docker-desktop-as-container-runtime).
+
+To fix this issue, please ensure that the `KIND_EXPERIMENTAL_PROVIDER=podman` is set properly. You should be able to see _enabling experimental podman provider_ while running kind commands or the scripts. If the issue remains even after this, please try to disable docker and try again.
+
+A final fix is to uninstall docker and use podman only.
+
+### Looping with "error: availableReplicas is not found"
+
+This issue is related to the Argo CD installation. It waits for the ingress controller to be ready before creating the ingress object for it. You can wait or if you don't need Argo CD you can pass `NO_ARGOCD=true` before running the script.
```

Comment:
This is not really an issue per se. It is the expected behavior as the script checks whether the ingress controller is available. It is expected to see a few lines like that with any installation. It is becoming an issue when the ingress controller does not become ready on time or never becomes ready.
I think it would be good to state first that a few lines like the one above are expected.

Reply:
Sure, I updated it.